adobe/react-spectrum
# Rac dropzone/filetrigger docs

mostly docs but there are some additional changes to dropzone and filetrigger.

- Text will no longer take the slot `heading` and instead will take `label`
- Removed FileTrigger's wrapper div -> this means no more context, no SlotProps, no DOMProps. The behavior of FileTrigger should still be the same though.


## ✅ Pull Request Checklist:


## 📝 Test Instructions:

- check to make sure the example cards look good
- make sure filetrigger/dropzone behaves as it did before

## 🧢 Your Project:
